languages.toml

```
@@ -3810,3 +3810,17 @@ language-servers = ["circom-lsp"]
 [[grammar]]
 name   = "circom"
 source = { git = "https://github.com/Decurity/tree-sitter-circom", rev = "02150524228b1e6afef96949f2d6b7cc0aaf999e" }
+
+[[language]]
+name = "nginx"
+scope = "source.nginx"
+injection-regex = "nginx"
+file-types = ["conf"]
+roots = []
+comment-token = "#"
+indent = { tab-width = 4, unit = "    " }
+auto-format = false
```

```suggestion
comment-token = "#"
indent = { tab-width = 4, unit = "    " }
```

these keys are set to their defaults so they can be removed